dev-tex/dvi2tty/dvi2tty-6.0.2.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Tool to show (La)TeX dvi files as text"
+DESCRIPTION="dvitty is intended for previewing dvi-files on text-only devices (terminals and \
+lineprinters). The output is not very pretty many times, but it still saves quite a lot of work, \
+especially if you have a little ways to walk to a laserprinter, for example.
+
+The program was originally written in Hedrick-Pascal, running on TOPS-20, and was later ported to \
+UNIX (BSD, Berkeley-pascal).
+
+It is not very smart in all situations, but still serves it pur- pose fairly well.
+
+For information on how to use see the man-page.
+
+Disdvi is a simple hack that dumps a dvi file in a more readable form.  It is not a spectacular \
+program but use it and improve as you wish. I'd appreciate any enhancements made, bug reports \
+etc. mailed to me."
+HOMEPAGE="https://github.com/t-tk/dvi2tty"
+COPYRIGHT="1989-2016 many, including Marcel J.E. Mol"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/t-tk/dvi2tty/archive/refs/tags/dvi2tty-$portVersion.tar.gz"
+CHECKSUM_SHA256="1386c17b8467858b2a5616cae63d811e2fba8392465548825e8ff0b9dd42bb1a"
+SOURCE_DIR="dvi2tty-dvi2tty-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dvi2tty$secondaryArchSuffix = $portVersion
+	cmd:disdvi
+	cmd:dvi2tty
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libkpathsea$secondaryArchSuffix
+	lib:libptexenc$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libkpathsea$secondaryArchSuffix
+	devel:libptexenc$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+TEST_REQUIRES="
+	cmd:diff
+	"
+
+defineDebugInfoPackage dvi2tty$secondaryArchSuffix \
+	$prefix/bin/disdvi \
+	$prefix/bin/dvi2tty
+
+BUILD()
+{
+	cd source/texk/dvi2tty
```

Comment:
This one uses the texlive build system (autotools) and not its "native" Makefile because that seems broken. The repository of this is a partial snapshot/fork of the texlive one, so I'm also not sure if it makes that much sense to split off this one.